appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml

```diff
@@ -0,0 +1,51 @@
+id: ebb81aaf055247c19579723cc84b8e74
+name: FreeBSD 10.3
+version: 1.0
+publisher: Trivago GmbH
+description: |-
+  Des:FreeBSD 10.3 with contextulization
+short_description: FreeBSD 10.3 with contextulization 
+
+tags:
+- freebsd
+- '4.8'
+- '4.10'
+- '4.12'
+- '4.14'
+- '5.0'
+
+# Other formats could be raw or vmdk
+format: qcow2
+
+creation_time: 1486408847
+os-id: FreeBSD
+os-release: '10.3'
+os-arch: x86_64
+hypervisor: KVM
+
+opennebula_version: 4.8, 4.10, 4.12, 4.14, 5.0
+
+opennebula_template:
+  context:
+    network: 'YES'
+    ssh_public_key: "$USER[SSH_PUBLIC_KEY]"
+    set_hostname: "$NAME"
+  cpu: '1'
+  graphics:
+    listen: 0.0.0.0
+    type: vnc
+  memory: '768'
+  os:
+    arch: x86_64
+  user_inputs:
+    ssh_user_key: "M|text|SSH KEY: Additional ssh key to add",
+
+images:
+- name: FreeBSD-10.3
+  url: https://one-trivago.s3-website-us-west-1.amazonaws.com/freebsd-trv.qcow2
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 21474836480
+  checksum:
+    md5: 365b676bf858246e1b68f48f54159ba0
```

Comment:
The checksum is wrong, I got:
md5: `8a49f2dac4ae37e25b366716d3a9c4cf`
sha256: `1a0dc8afbb11a1759d02949472b420c800e7e967aa3147d408862424b6cf552f`
Can you please check?